common/app/views/fragments/head.scala.html

```diff
@@ -26,7 +26,7 @@
 
 @if(mvt.PolyfillIO.isParticipating) {
     <link rel="prefetch"
-          href="https://polyfill.guim.co.uk/v2/polyfill.min.js?features=es6,Element.prototype.closest,Element.prototype.matches">
+          href="https://assets.guim.co.uk/polyfill-io/v2/polyfill.min.js?features=es6,Element.prototype.closest,Element.prototype.matches">
```

Comment:
also it looks like the regex we're using is not escaping the full stop in `polyfill.io` - allowing `polyfill-io` to work.  Probably should avoid this and stick to `polyfill.io`